tests/webidl/test.idl

```diff
@@ -92,6 +92,18 @@ interface Inner {
   [Operator="+="] void incInPlace([Const, Ref] Inner i);
 };
 
+[Prefix = "Space::"]
+interface InnerUserBase
```

Comment:
please follow the style in the rest of the test, which is java-style `{` on the previous line.

Reply:
fixed